[SG-1971] -- Added a field for mastodon social media link on agencies

paboden opened the pull request:
[SG-1971]

Added a field for mastodon social media link on agencies

Instructions:
- Clear cache
- Config import
- Edit an agencies social media content and add mastadon field value.

[SG-1971]: https://sfgovdt.jira.com/browse/SG-1971?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

skinnylatte commented:
Testing this.

### Steps

1. [Added test website](https://pr-1532-sfgov.pantheonsite.io/departments/gi-test-department) to Mastodon staging account
2. Refreshed profile

### Result

Verification not working: (usually appears immediately)
 
<img width="588" alt="Screenshot 2023-03-31 at 11 01 58 AM" src="https://user-images.githubusercontent.com/2878228/229195946-816f2b00-003f-4ee0-bc83-50eb8088bdfa.png">

### Troubleshooting

- is the link correct in the dept page?
- looks okay
- need to dig into it more

<img width="413" alt="Screenshot 2023-03-31 at 11 09 39 AM" src="https://user-images.githubusercontent.com/2878228/229197424-8cb07295-ff1a-4ee2-abbf-70c124146452.png">

paboden commented:
Hi @skinnylatte 

https://pr-1532-sfgov.pantheonsite.io/departments/gi-test-department

I’ve removed the 'noopener' and 'target' from mastadon social media link. I’ve also added the verification meta/link information to the head tag on the page (recommended here - https://docs.joinmastodon.org/user/profile/#verification ).

It is very possible that pantheons network/IP resolution/PR-domain system may be causing the verification to fail. There is a good chance that it will work on prod, but realistically I can’t guarantee that. @arlina-espinoza  might be able to tell us more, as she is more aware of the details of pantheons setup. Or we may need to open a support ticket with pantheon and get their feedback.

Cheers
Phil
